backend/lib/services/controllerregistrations.js

```diff
@@ -30,3 +30,14 @@ exports.list = async function ({ user }) {
   const controllerregistrations = getControllerRegistrations()
   return fromResource(controllerregistrations)
 }
+
+exports.listNetworkingTpes = async function () {
+  const controllerregistrations = getControllerRegistrations()
+  const networkExtensions = _.filter(controllerregistrations, ({ spec }) => {
+    return _.find(spec.resources, { kind: 'Network' })
+  })
+
+  return _.flatMap(networkExtensions, ({ spec }) => {
+    return _.map(spec.resources, 'type')
+  })
```

Comment:
```suggestion
  return _
    .chain(controllerregistrations)
    .map('spec.resources')
    .flatMap()
    .filter(['kind', 'Network'])
    .map('type')
    .value()
```

Reply:
ah yes, good point to use chain. I moved this from the frontend where we cannot use chaining and forgot to change :)